t1.py

```diff
@@ -0,0 +1,42 @@
+import mmtapi.mmtapi as mmtapi
+
+demoTOKEN = '0f3732fcfb1c8577c390e71a176f3c8f'
```

Comment:
You shouldn't commit ANY tokens to branch.

implement reading from environmental variables os.environ.get('demoTOKEN')